rust/src/main.rs

```diff
@@ -177,6 +177,33 @@ mod tests {
         true,
         (3, 8)
     )]
+    #[case::subsubtable(
+        indoc ! {r#"
+    [project]
+    [tool.coverage.report]
```

Comment:
Please also add a tool.Coverage entry here that should be reordered first within this group.